src/test/java/org/numenta/nupic/RunLayer.java

```diff
@@ -84,8 +89,8 @@
         private static List<int[]> input;
         private static List<String> raw;
         
-        private static String encFilePath = "/Users/cogmission/git/lscheinkman/NAB/art_daily_encoder_output_java.txt";
-        private PrintWriter encFile = null;
+//        private static String encFilePath = "/Users/cogmission/git/lscheinkman/NAB/art_daily_encoder_output_java.txt";
+//        private PrintWriter encFile = null;
```

Comment:
Hi Matt,

I love you man! Thanks for having a look! 

Yes, this is a workhorse file not intended to be anything other than a scratchpad utility. I left the comments because I need to see the configurations I used for other outputs.

So I have:

* Configurations I used to do SP-only output
* Configurations I used to do TM-only output
* Configurations I used to do Encoder-only output
* Configurations I used to vary the input files...

Each "type" of output or configuration had certain inputs and settings - the combination of which I couldn't commit to memory and so leaving in the comments made (makes) it easier 
for me to flit back and forth between them as new debugging ideas occurred to me...